.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -29,22 +30,24 @@ guidelines at https://www.apache.org/security/ regarding vulnerability disclosur
 -  Traffic Router
 -  Traffic Stats
 -  Traffic Vault
+-  CI tests
 -  unknown
 
 ## Current behavior:
-<!-- Describe how the bug manifests / how the current features are insufficient. -->
+<!-- Describe how the bug manifests -->
 
-## Expected / new behavior:
-<!-- Describe what the behavior would be without the bug / how the feature would improve Traffic Control -->
+## Expected behavior:
+<!-- Describe what the behavior would be without the bug -->
 
 ## Minimal reproduction of the problem with instructions:
 <!--
-If the current behavior is a bug or you can illustrate your feature request better with an example,
-please provide the *STEPS TO REPRODUCE* and include the applicable TC version.
+If the current behavior is a bug, please provide the *STEPS TO REPRODUCE* and
+include the applicable TC version.
 -->
 
 ## Anything else:
-<!-- e.g. stacktraces, related issues, suggestions how to fix -->
+<!-- e.g. stacktraces, related issues, suggestions how to fix (feel free to
+delete this section) -->
```

Comment:
nit: with multi-line comments, I find it easier to parse when the `<!--` and `-->` are on separate lines like this:
```
<!--
This is an
example multi-line
comment.
-->
```

Sometimes I find myself accidentally changing text inside a comment (usually the "what does this PR do" section).

Reply:
Put `<!--` and `-->` on different lines for multiline comments in 1072cb9d5f.